docusaurus.config.js

```diff
@@ -119,5 +120,22 @@ module.exports = {
         pages: false,
       },
     ],
+    [
+      'redocusaurus',
+      {
+        // Plugin Options for loading OpenAPI files
+        specs: [
+          {
+            spec: 'https://raw.githubusercontent.com/authzed/authzed-go/main/proto/apidocs.swagger.json',
```

Comment:
It would be nice if we can hide the empty services in the generated docs. I'll see if the plugin has something to do that.

Reply:
Generally these HTTP docs don't integrate well enough that it looks professional.
I'd almost prefer to just link out to swagger hub or something else similar to our buf registry link.

It replaces the nav on the left with no indication of how to go back and the additional search bar it adds is near useless.

When we built Quay's docs we could even save OAuth tokens locally so people could test against prod: https://docs.quay.io/api/swagger/#!/build/cancelRepoBuild

Reply:
Agree and after looking into configuring redoc/redocusaurus, we don't have a good way of hiding the empty services. I'd prefer pointing a subdomain at swagger hub and linking to that instead.